pages/about.js

```diff
@@ -0,0 +1,119 @@
+//global imports
+import React from "react";
+import AboutLayout from "../src/components/layouts/AboutLayout";
+import { Typography, Container } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+//data
+import about_page_info from "../public/data/about_page_info";
+import members from "../public/data/members.json";
+import links from "../public/data/links.js";
+//local components
+import AboutHeaderImage from "../src/components/about/AboutHeaderImage";
+import InfoBubble from "../src/components/about/InfoBubble";
+import InfoLink from "../src/components/about/InfoLink";
+import Member from "../src/components/about/Member";
+
+const useStyles = makeStyles(theme => {
+  return {
+    centeredText: {
+      textAlign: "center"
+    },
+    textBlock: {
+      paddingTop: theme.spacing(2),
+      paddingBottom: theme.spacing(1)
+    },
+    sectionHeadline: {
+      marginBottom: theme.spacing(6)
+    },
+    bubbleGrid: {
+      padding: 0,
+      maxWidth: 1390,
+      margin: "0 auto",
+      textAlign: "center"
+    },
+    memberGrid: {
+      maxWidth: 1390,
+      margin: "0 auto",
+      textAlign: "center",
+      verticalAlign: "top",
+      padding: 0
+    },
+    mainFocuses: {
+      width: 800,
+      margin: "0 auto"
+    }
+  };
+});
```

Comment:
Not entirely sure about the best practice so correct me if I am wrong. But should we put the styles const into a separate file? Maybe we can have styles directory and then import files from that directory. cc/ @EvanHahn

Reply:
I feel like it's easier to work with this way

Reply:
Yeah, I am fine with it. Just wanted to confirm.